odo/backends/sql_csv.py

```diff
@@ -19,6 +22,8 @@
 
 from ..append import append
 from ..convert import convert
+from ..compatibility import StringIO
+from ..utils import literal_compile
 from .csv import CSV, infer_header
 from ..temp import Temp
 from .aws import S3
```

Comment:
Unrelated, but this import appears to be unused, since there's a try/except for importation farther down.